background.js

```diff
@@ -71,13 +72,20 @@ function bufferData(tabId, message) {
   }
 
   if (message.event === 'scope:new') {
-    tabData.scopes[message.data.child] = {
```

Comment:
So, this was used to "hydrate" the Scopes view when the DevTools would open, at which point the DevTools app would take over maintaining the scopes info. With my incorrect change, the scopes would not be hydrated when you opened DevTools, so you needed to refresh the app with the DevTools open.

Thx for watching out for my blunders :smiley:
It really sucks that there are no substancial tests. Hoping to fix it some time.